public/stylesheets/style.scss

```diff
@@ -1,8 +1,135 @@
-body {
-  padding: 50px;
+$sand-top: #D7D3C8;
+$sand-bottom: #F6F4EC;
+$salmon: #D9615B;
+$science-blue: #C9E3F6;
+$spacer-orange: #F7C4A6;
+
+html, body {
   font: 14px "Lucida Grande", Helvetica, Arial, sans-serif;
+  margin:0;
+  padding:0;
+  height:100%;
 }
+body{
+	background-image: url(/images/grain.png), (linear-gradient(white, white));
 
+}
+.main{
+	//background-image: url(/images/grain.png), (linear-gradient($sand-top, $sand-bottom));
+	height:100%;
+	width:100%;
+}
+#moz-logo{
+	margin:0;
+	padding:0;
+
+}
+#navbar-brand-link{
+	margin:0;
+	padding:0;
+	margin-top:10%;
+}
+#form{
+	margin-top:50px;
+}
+#footer-background{
+	height:373px;
+	background-color: $salmon;
+	border-top: 12px solid $science-blue;
+}
+#footer-image{
+	height:127px;
+}
+#footer{
+	height:500px;
+}
+#moz-footer-logo{
+	margin-bottom:10px;
+}
+#footer-text{
+	width:90%;
+	margin:0 auto;
+}
 a {
   color: #00B7FF;
 }
+.navbar{
+	margin-bottom:0px !important;
```

Comment:
if specificity is the issue, it'd be awesome if we could use ID's here as opposed to the evil `!`